config/example_ros_configuration.yaml

```diff
@@ -0,0 +1,12 @@
+ueye_cam:
```

Comment:
I'm fine with this configuration, but I could see the argument for having an example with more parameters. This is not a stopper for this PR.

Reply:
I left it short (with relevant comments) to highlight that the defaults are in fact a useful basis to start from. I think, for me, this is the minimal set of parameters you'd use to get a basic colour stream &rarr; your hello world for ueye_cam, so to speak.

Added an informative comment providing some context at the top of the yaml. Will this do?

Reply:
Yep, looks good.